### ISSUE TITLE
Anonymous field instead of interface

### ISSUE THREAD

**Issue by nrgjack** [madskristensen/TypeScriptDefinitionGenerator#7]
### Installed product versions
- Visual Studio: VS 2017 RC 
- This extension: 1.0.11

### Description
When generating a class property the resultins ts is not referring to the ts generated interface referring to the class

### Steps to recreate
1. Create an Address.cs class
```
    public class Address 
    {
        public string  Street{ get; set; }
    }
```
2. Enable TS Generation on Address.cs
3. Create a Person.cs class 

```
    public class Person
    {
        public string  Name { get; set; }
        public Address HomeAddress { get; set; }
    }
```
4. Enable TS Generation on Person.cs

### Current behavior
Address.d.ts
`
declare module server {
	interface address {
		street: string;
	}
}
`
Person.d.ts
`
declare module server {
	interface person {
		name: string;
		homeAddress: {
			street: string;
		};
	}
}
`

### Expected behavior
Address.d.ts
`
declare module server {
	interface address {
		street: string;
	}
}
`
Person.d.ts
`
declare module server {
	interface person {
		name: string;
		homeAddress: address;
	}
}
`

**Comment by idg10** [comment]
This appears to happen because the code that checks whether the relevant type has had TypeScript generation enabled (`IntellisenseParser.HasIntellisense`) is looking for names of the form `Something.cs.d.ts` (which is the naming convention that the VS 2015 Web Essentials used) but this new extension creates files called `Something.d.ts` (note the missing `.cs`) - a change reported in #13 

Reverting to the old file naming convention would fix this problem. Alternatively you could update `IntellisenseParser.HasIntellisense` to be consistent with the new naming convention - that would also fix the problem. (So it really depends on whether the change reported in #13 was deliberate.)

**Comment by raboud** [comment]
Please Try Build 1.1.24 to see if your issues are corrected.

**Comment by nhammadi** [comment]
@madskristensen This issue was closed with 1.1.24 release, you should close it
